doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+ 
 
-The basic approach to producing the initial deliverables is to:
-A) Summarize the overall CNTT Networking and Fabric Strategy
-B) Document the initial list of Objectives and Requirements
-C) Document enough initial logistical details for contributors to create coherent content
+<a name="5.1.1"></a>
+### 5.1.1 Networking Scope
+According to the ETSI NFV Model, Networking alongside Compute and Storage, is an integral part of the Network Function Virtualisation Infrastructure (NFVI). The general function of Networking in the NFV context is to provide the connectivity between various virtualised and non-virtualised resources required for a delivery of a Network Service. Such a connectivity may manifest itself as a virtualised network between VMs and/or Containers (e.g. overlay networks managed by SDN Controllers, and/or programmable networking fabrics that provide such connectivity natively) or as an integration into the Infrastructure Hardware level for offloading some of the Network Functions into this level (accelerated networking functions). 
 
-Readers be aware, in parallel with the CNTT Networking Strategy, the RI team is implementing networking for use today in labs and by RC. As RI has to deliver a working network at the same time the initial networking strategy and specifications are to be delivered, it is not expected nor mandated RI networking be conformant in the immediate future. However, the RI will need to be compatible, meaning, it delivers the network connectivity required by the RA and by RC, even if it does not implement the APIs, topology, encapsulation, etc., that will ultimately be specified herein.
+Normalization of the integration reference points between different layers of the NFV architecture stack is one of the main concerns of the telco industry.  In the Networking context the primary focus is directed on the packet flow and control flow interfaces between Virtualisation Layer (be it IaaS or CaaS type of virtualisation) and Networking Hardware resources, as well as on related integration into the various MANO reference points (Hardware/ Network Infrastructure Management/Control, Orchestration). 
 
-The following subsections, Executive Summary, Strategy Objectives, Networking Requirements and Initial Approach, respectively, are intended to provide an overview of CNTT's vision for networking and intent, objectives, requirements and their supporting rationale, as well as initial development approach.
+It is an important feature of the proposed Networking Model to ensure the ability for supporting simultanous usage of different reference architectures by having multiple CaaS and IaaS layers on top of a shared Hardware Infrastructure layer. The details of this approach are provided in the section "Network Layering and Concepts" below.
```

Comment:
@ulikleber I feel, as you say,  the content call for an RM model that allow simultanous implementations following same and/or differnt RA to coexist without going back to the ages when all HW infrastructure was private to each application. Would you have an alternative formulation to state this?

Reply:
@ulikleber Separate layers in Cloud Infrastructure (aka NFVI)  are a fact of life, as you know from any implementation.  What we in CNTT are working on is how to standardize network infrastructure in such a way that it supports migration from IaaS to CaaS and also simultaneous implementation of both types of virtualization, all using the same network fabric.  This is a common and very important problem in practical implementations and needs to be addressed by CNTT, even if it is an additional complexity. Otherwise, this complexity has to be dealt by operators in a non-standard way, as it happens now.  Btw:  we need also address connectivity to the edge sites and to public clouds as well, and we want to use the same networking fabric whenever possible. To @TFredberg point, what is a better formulation of this problem, as I believe all of us recognize this as a real problem?

Reply:
@ulikleber @walterkozlowski  @TFredberg The key message in this paragraph is that the "**what**" - that we "ensure the ability for supporting simultaneous usage of different reference architectures ", not the how. For this last paragraph, I suggest we just remove the how, so readers focus on the what. 

* "simultanous" - Spelling

* "It is an important feature of..." - I'd suggest we amend this to read "It is a **_critical_** feature of...".

Reply:
Agree with @markshostak . My goal in the comment was not to ignore layered solutions. But the "important feature" is to provide resources for containerized workloads and for hypervisors by one cloud infrastructure. This can be done by layers or other means.

Reply:
I disagree that we can omit the parts that needs to be shared in between simultaneous implementations based on RA-1 and RA-2.
This document is mainly discussing Networking, and practically, which is what CNTT is there to specify, parts needs to be shared of the Networking Underlay. This needs to be possible to operate (manage) from an administrative domain that can be keep separate from the SW Virtualization Infrastructure Implementations administrative domains.
I can not see any other way to model this then by logical layers of management. This however do not have to be the same as a layered realization.

Reply:
@TFredberg Hi Tomas, Just to clarify, my intent was to remove the "how" _from the Networking Scope_, not from the document entirely. IMHO the scoping section should focus on _what_ needs to be done, not _how_ to do it. I'd still suggest removing the lines in question, and addressing the topic in a more appropriate section. Thx, -M